editor/action_map_editor.cpp

```diff
@@ -453,10 +453,14 @@ void ActionMapEditor::update_action_list(const Vector<ActionInfo> &p_action_info
 			// First Column - Icon
 			Ref<InputEventKey> k = event;
 			if (k.is_valid()) {
-				if (k->get_physical_keycode() == Key::NONE) {
+				if (k->get_physical_keycode() == Key::NONE && k->get_keycode() == Key::NONE && k->get_key_label() != Key::NONE) {
+					event_item->set_icon(0, action_tree->get_theme_icon(SNAME("KeyboardUnicode"), SNAME("EditorIcons")));
```

Comment:
Should the icon be renamed to `KeyboardKeyLabel` for consistency?

Reply:
Renamed.